src/main/scala/cromwell/webservice/WorkflowQueryKey.scala

```diff
@@ -23,7 +23,7 @@ object WorkflowQueryKey {
 
   case object Name extends SeqStringWorkflowQueryKey {
     override val name = "Name"
-    private val WorkflowNamePattern = "([a-zA-Z][a-zA-Z0-9_]+)".r
+    private val WorkflowNamePattern = "([a-zA-Z][a-zA-Z0-9_]*)".r
```

Comment:
I thought perhaps we could reach into the `WdlParser.java` and pull out this regex, but it seems like it might be more trouble than it's worth right now